src/main/java/seedu/duke/commands/HelpCommand.java

```diff
@@ -67,7 +67,7 @@ public CommandResult execute(ModuleList moduleList, Configuration configuration)
         case TAG_COMMAND_WORD:
             return new CommandResult(TAG_HELP);
         case OPTION_COMMAND_WORD:
-            return new CommandResult(OPTION_HELP + "\n" + configuration.getConfigurationGroupExplain());
```

Comment:
I don't think we should delete the list of group names and the explanations, otherwise, users may not understand what each group means clearly

Reply:
I thought each group name is already explained in the `option GROUP_NAME` command? Maybe, instead of making the user type every option name to find out what it does, we should list the name + explanation as part of the `option` command (no arguments) instead of just the name.